server/tests/test_restart.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import pytest
+from channelfinder import ChannelFinderClient
+from testcontainers.compose import DockerCompose
+
+from .client_checks import channels_match, check_channel_property, create_client_and_wait, wait_for_sync
+from .docker import restart_container, setup_compose  # noqa: F401
+
+PROPERTIES_TO_MATCH = ["pvStatus", "recordType", "recordDesc", "alias", "hostName", "iocName", "recceiverID"]
+
+LOG: logging.Logger = logging.getLogger(__name__)
+
+EXPECTED_DEFAULT_CHANNELS = 32
+
+
+@pytest.fixture(scope="class")
```

Comment:
You could also do module scope and skip the class, since it's only two tests in this file